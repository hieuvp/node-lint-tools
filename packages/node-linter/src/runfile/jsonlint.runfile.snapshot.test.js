const { parseJSONLintArgs } = require('./jsonlint.runfile');

[
  '.',
  'packages/',
  'packages/hapi-linter/',
  'packages/node-linter/',
  'packages/react-linter/',
  'packages/react-native-linter/',
  'runfile.js',
  'README.md',
  'package.json',
  'packages/hapi-linter/package.json',
  'packages/node-linter/src/runfile/blacklist.runfile.json'
].forEach(args => {
  it(`should match snapshot when parsing "${args}"`, () => {
    expect(parseJSONLintArgs(args)).toMatchSnapshot();
  });
});
