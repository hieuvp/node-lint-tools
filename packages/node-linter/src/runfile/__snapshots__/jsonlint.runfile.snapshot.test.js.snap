// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot when parsing "." 1`] = `
Array [
  "packages/hapi-linter/src/eslintrc/hapi.rules.eslintrc.json",
  "packages/node-linter/src/eslintrc/node.dev-rules.eslintrc.json",
  "packages/node-linter/src/eslintrc/node.rules.eslintrc.json",
  "packages/node-linter/src/runfile/blacklist.runfile.json",
  "packages/react-linter/src/eslintrc/react.rules.eslintrc.json",
  "packages/react-native-linter/src/eslintrc/react-native.rules.eslintrc.json",
]
`;

exports[`should match snapshot when parsing "README.md" 1`] = `Array []`;

exports[`should match snapshot when parsing "package.json" 1`] = `
Array [
  "package.json",
]
`;

exports[`should match snapshot when parsing "packages/" 1`] = `
Array [
  "packages/hapi-linter/src/eslintrc/hapi.rules.eslintrc.json",
  "packages/node-linter/src/eslintrc/node.dev-rules.eslintrc.json",
  "packages/node-linter/src/eslintrc/node.rules.eslintrc.json",
  "packages/node-linter/src/runfile/blacklist.runfile.json",
  "packages/react-linter/src/eslintrc/react.rules.eslintrc.json",
  "packages/react-native-linter/src/eslintrc/react-native.rules.eslintrc.json",
]
`;

exports[`should match snapshot when parsing "packages/hapi-linter/" 1`] = `
Array [
  "packages/hapi-linter/src/eslintrc/hapi.rules.eslintrc.json",
]
`;

exports[`should match snapshot when parsing "packages/hapi-linter/package.json" 1`] = `
Array [
  "packages/hapi-linter/package.json",
]
`;

exports[`should match snapshot when parsing "packages/node-linter/" 1`] = `
Array [
  "packages/node-linter/src/eslintrc/node.dev-rules.eslintrc.json",
  "packages/node-linter/src/eslintrc/node.rules.eslintrc.json",
  "packages/node-linter/src/runfile/blacklist.runfile.json",
]
`;

exports[`should match snapshot when parsing "packages/node-linter/src/runfile/blacklist.runfile.json" 1`] = `
Array [
  "packages/node-linter/src/runfile/blacklist.runfile.json",
]
`;

exports[`should match snapshot when parsing "packages/react-linter/" 1`] = `
Array [
  "packages/react-linter/src/eslintrc/react.rules.eslintrc.json",
]
`;

exports[`should match snapshot when parsing "packages/react-native-linter/" 1`] = `
Array [
  "packages/react-native-linter/src/eslintrc/react-native.rules.eslintrc.json",
]
`;

exports[`should match snapshot when parsing "runfile.js" 1`] = `Array []`;
